account_bank_match/data/account_bank_match_rule.xml

```diff
@@ -170,5 +170,17 @@
             <field name="type">bonus</field>
         </record>
 
+        <record
+            id="account_bank_match.account_bank_match_rule_18"
+            model="account.bank.match.rule">
+            <field name="active">True</field>
+            <field name="rule">[('partner_invoice_id.country_id', '=', '@_get_iban_country_id())@')]</field>
+            <field name="model">sale.order</field>
+            <field name="score_item">10</field>
+            <field name="score">0</field>

```

Comment:
Will this rule have any effect when score is 0?

Reply:
@NL66278 Changed the type of the rule to `bonus` so now it will not have any effect. Either it is True and 10 points are added of false and nothing is added to the end score